src/pages/createCohort/components/cohortList.jsx

```diff
@@ -17,6 +18,24 @@ const CohortList = () => {
       .catch(console.log);
   }, []);
 
+  if (header) {
+    return (
+      <div className="cohort-teacher-view">
+        {cohorts.map(cohort => {
+          if (cohort.name !== null) {
+            return (
+              <Card className="cohort-view">
+                {`id - ${cohort.id}`}
+                {` | Name - ${cohort.name}`}
+              </Card>
+            );
+          } else {
+            return <Card className="cohort-view">{`id - ${cohort.id}`}</Card>;
+          }
+        })}
```

Comment:
Every component that renders from a map like this will need a `key` property so React can keep track of things under the hood. You can do this by changing the signature of the .map to 

```js
.map((cohort, index) => {
    // code in here
})
```

Then on the components you render inside that map, add a `key` property and give it the `index` parameter that gets passed in to the map callback: `<Card key={index} className="cohort-view"> </Card>`

Reply:
OK, went and read up a bit more in the react docs. Added 'cohort.id' as the keys. Does that work OK?